session/action.php

```diff
@@ -0,0 +1,47 @@
+<?php
+session_start();
+/**
+ * Hard-coding the username and passwords
+ */
+$username = 'sayan';
+$password = '1234';
+
+if (isset($_POST["submit"])) {
+  /**
+   * checking if the username and passwords are  valid else go back to login page after showing alert
+   */
+  if ($username != $_POST['username'] || $password != $_POST['password']) {
```

Comment:
If user name and password do not match, then redirect to login page.

Reply:
@Sayan-Innoraft Where did this file go?

Reply:
I renamed the file to homepage.php

Reply:
Ok